### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+project: fickling
```
Comment (xansec) [inline Mayhemfile:1]: Can we change the target name to `fuzz_pickle_decompiler`?

**Review by xansec — CHANGES_REQUESTED** [review]
Is it necessary to upload the corpus files? Usually 1-2 files suffice. Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (ebell495) [reply Mayhemfile:1]: I fixed this

**Comment by ebell495** [comment]
I've removed most of the fuzz corpus, and it seems to work with just 3

**Pull request merged**